src/garage/sampler/_dtypes.py

```diff
@@ -81,6 +81,7 @@ def to_batch(self):
                                                   dtype=StepType),
                             env_infos=env_infos,
                             agent_infos=agent_infos,
+                            episode_infos=dict(),
```

Comment:
you need to modify this class to track episode_infos as well -- it is used to create EpisodeBatches in the sampler. If they aren't captured here, they will be lost.